amy/static/scss/custom_bootstrap.scss

```diff
@@ -0,0 +1,98 @@
+// custom_bootstrap.scss
+
+// compile with Sass:
+// $ npm install -g sass
```

Comment:
I'd prefer to put `sass` as dev dependency in `package.json`. This way everyone following readme will have it installed locally if they need to regenerate this file.